Xamarin.Forms.Core/BindableObject.cs

```diff
@@ -57,7 +57,9 @@ public bool IsSet(BindableProperty targetProperty)
 			if (targetProperty == null)
 				throw new ArgumentNullException(nameof(targetProperty));
 
-			return GetContext(targetProperty) != null;
```

Comment:
I think, with the modified behavior, that `IsDefault()` was a better name. It's not too late to make that change...

Reply:
When I see a method here that says "IsDefault", I'm going to expect it to return true if the value == the default value, whether that's set by the prop or set by a default value creator or set by my code. This gets us part of that behavior but not all of it, I think. Correct me if I'm wrong.

Reply:
I like to think that a v value created with the `defaultValueCreator` is not a default value, as it should be read, and it's value honored.

@jassmith could you weight in one name or the other ?